include/config/config_collision.h

```diff
@@ -17,6 +17,7 @@
 
 /**
  * Allows all surfaces types to have force, (doesn't require setting force, just allows it to be optional).
+ * Also allows you to pass a warp node to SURFACE_WARP floors via the second byte of the force parameter.
```

Comment:
But also the death plane and vertical wind floors.

How is this actually being enforced anyway, is it pretty must just automatically handled by the nature of this define?

Reply:
idk what this means but it just checks if force isn't 0 and if a surface doesn't have force it defaults to 0

Reply:
fixed